dockerfiles/Dockerfile.server

```diff
@@ -0,0 +1,18 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+# Official user quickstart docker container for ONNX Runtime
+# Ubuntu 16.04, CPU version, Python 3.
+#--------------------------------------------------------------------------
+
+FROM ubuntu:16.04
+MAINTAINER Harry Yang "huayang@microsoft.com"
+
+WORKDIR /onnxruntime/server/
+ENV MODEL_ABSOLUTE_PATH /onnxruntime/model/model.onnx
+COPY ./onnxruntime_server /onnxruntime/server/
```

Comment:
Where is the 'onnxruntime_server' file? How did you get it?

Reply:
yes, where does this binary come from and how does one know what flavor/version which was deployed?
btw, does onnxruntime_server have any dependencies?

Reply:
agreed I would like the container to build the project

Reply:
Our initial idea was to build the docker image from the directory that contains our server "onnxruntime_server" to skip the build step and make the docker as simple as possible. But seems you guys prefer adding the build version so I added the build step.
The only dependency onnxruntime_server needs on Ubuntu 16.04 is "libgomp1" which is installed in the Dockerfile.